src/components/GlobalPreferences/Notifications/notification-service-api.js

```diff
@@ -11,7 +11,7 @@ import {
 
 // The notifications API expects mainnet or rinkeby. This deviates from web3's getNetworkType which returns main
```

Comment:
let's remove web3's getNetworkType and replace it with use-wallet's networkType.